ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -840,7 +864,7 @@ func isPIInstanceRefreshFunc(client *st.IBMPIInstanceClient, id, instanceReadySt
 			return nil, "", err
 		}
 		// Check for `instanceReadyStatus` health status and also the final health status "OK"
-		if *pvm.Status == helpers.PIInstanceAvailable && (pvm.Health.Status == instanceReadyStatus || pvm.Health.Status == helpers.PIInstanceHealthOk) {
+		if (*pvm.Status == helpers.PIInstanceAvailable || *pvm.Status == "SHUTOFF") && (pvm.Health.Status == instanceReadyStatus || pvm.Health.Status == helpers.PIInstanceHealthOk) {
```

Comment:
Let's not complicate this condition more.
Add a line before this to check the SHUTOFF status and return.

Reply:
Add a separate check but It doesn't make any difference, if anything it is just redundant.